docs/essentials/numericsincore.md

```diff
@@ -1 +1,58 @@
+[System.UInt64](http://dotnet.github.io/api/System.UInt64.html) | Unsigned | 8 | 0 | 18,446,744,073,709,551,615
+
+Each integral type supports a standard set of arithmetic, comparison, equality, explicit conversion, and implicit conversion operators. Each integer also includes methods to perform equality comparisons and relative comparisons, to convert the string representation of a number to that integer, and to convert an integer to its string representation. Some additional mathematical operations beyond those handled by the standard operators, such as rounding and identifying the smaller or larger value of two integers, are available from the [System.Math](http://dotnet.github.io/api/System.Math.html) class. You can also work with the individual bits in an integer value by using the [System.BitConverter](http://dotnet.github.io/api/System.BitConverter.html) class. 
+     
+Note that the unsigned integral types are not CLS-compliant. For more information, see [Language Independence and Language-Independent Components](essentials\languageIndependence.md).
+
+## Floating-point types
+
+.NET Core includes three primitive floating point types, which are listed in the following table. 
+
+Type | Size (bytes) | Minimum Value | Maximum Value
+---- | ------------ | ------------- | -------------
+[System.Double](http://dotnet.github.io/api/System.Double.html) | 8 | -1.79769313486232e308 | 1.79769313486232e308
+[System.Single](http://dotnet.github.io/api/System.Single.html) | 4 | -3.402823e38 | 3.402823e38
+[System.Decimal](http://dotnet.github.io/api/System.Decimal.html) | 8 | -79,228,162,514,264,337,593,543,950,335 | 79,228,162,514,264,337,593,543,950,335
+   
+Each floating-point type supports a standard set of arithmetic, comparison, equality, explicit conversion, and implicit conversion operators. Each also includes methods to perform equality comparisons and relative comparisons, to convert the string representation of a floating-point number, and to convert a floating-point number to its string representation. Some additional mathematical, algebraic, and trigonometric operations are available from the **Math** class. You can also work with the individual bits in **Double** and **Single** values by using the **BitConverter** class. The **Decimal** structure has its own methods, **Decimal.GetBits** and **Decimal.Decimal(Int32())**, for working with a decimal value's individual bits, as well as its own set of methods for performing some additional mathematical operations. 
+
+The **Double** and **Single** types are intended to be used for values that by their nature are imprecise (such as the distance between two stars in the solar system) and for applications in which a high degree of precision and small rounding error is not required. You should use the **Decimal.GetBits** type for cases in which greater precision is required and rounding error is undesirable.
```

Comment:
Something like:

> The **Double** and **Single** types are intended to be used for values that by their nature are imprecise (such as the distance between two stars in the solar system) and for applications in which a high degree of precision is not required and small rounding error is acceptable. The **Decimal** type can accurately represent [decimal fractions](https://en.wikipedia.org/wiki/Decimal_fraction) (like monetary values) and it also generally has greater precision than the other floating-point types.

Though the claim that `decimal` can accurately represent any decimal fraction in incorrect (it's still limited by the size of its mantissa), I think it's a reasonable simplification for this document.